README.md

```diff
@@ -203,8 +203,17 @@ Did everything work as expected? Great - now onto the production site.
 Make sure your changes are deployed to the production server (i.e. you've cut a release with your changes).
 
 1. Look at the times cron tasks are run (specified in [`scripts/lametro-crontasks`](https://github.com/datamade/la-metro-councilmatic/blob/master/scripts/lametro-crontasks)), and plan to rebuild the index inbetween those times. Rebuilding the index will take a few minutes, so plan accordingly.
-2. Merge in the pull request and deploy.
+2. Deploy your changes to production.
```

Comment:
Sorry, my last comment was unclear: we do not need any part of Step 2 (cf. line 203 above). You can delete it, and renumber the subsequent steps.

Reply:
Right, because we already merged it into master...d'oh